core/src/main/scala/cromwell/core/WorkflowMetadataKeys.scala

```diff
@@ -20,5 +20,8 @@ object WorkflowMetadataKeys {
   val SubmissionSection_Inputs = "inputs"
   val SubmissionSection_Options = "options"
   val SubmissionSection_Imports = "imports"
+  val SubmissionSection_WorkflowType = "workflowType"
```

Comment:
Yes, this is [WES](https://github.com/ga4gh/workflow-execution-schemas/blob/develop/swagger/proto/workflow_execution.swagger.json#L409) terminology.